soc/arc/snps_nsim/Kconfig.defconfig.hs

```diff
@@ -13,7 +13,7 @@ config NUM_IRQ_PRIO_LEVELS
 
 config NUM_IRQS
 	# must be > the highest interrupt number used
-	default 20
+	default 30
```

Comment:
1. How this commit is related to "Direct IRQ" pull-request? I'd say it is completely orthogonal so let's move it to a separate pull-request.
2. I guess you meant "Number of interrupts available" but not "interrupt numbers".

Reply:
After implementing direct IRQ support, nsim_hs will fail on compiling for the tests/kernel/gen_isr_table because of wrong setting in NUM_IRQs.  So  I think it shoud be part of this pull request.

Reply:
That's OK then but in that case it's good to explain that in the commit message so it's immediately clear what are this change relationships with the main feature of the pull-request.

Reply:
done